Editor/Resources/EditorWindow/GameLiftPlugin.cs

```diff
@@ -0,0 +1,159 @@
+        {
+            GetWindow();
+        }
+
+        [MenuItem("Amazon GameLift/Bring Panel to Front", priority = 1)]
+        public static void FocusPanel()
+        {
+            ShowWindow();
+        }
+
+        [MenuItem("Amazon GameLift/Set AWS Account Profiles", priority = 100)]
+        public static void OpenAccountProfilesTab()
+        {
+            GetWindow().OpenTab("Credentials");
+        }
+
+        [MenuItem("Amazon GameLift/Host with Anywhere", priority = 101)]
+        public static void OpenAnywhereTab()
+        {
+            GetWindow().OpenTab("Anywhere");
+        }
+
+        [MenuItem("Amazon GameLift/Host with Managed EC2", priority = 102)]
+        public static void OpenEC2Tab()
+        {
+            GetWindow().OpenTab("EC2");
+        }
+
+        [MenuItem("Amazon GameLift/Import Sample Game", priority = 103)]
+        public static void ImportSampleGame()
+        {
+            string filePackagePath = $"Packages/{Paths.PackageName}/{Paths.SampleGameInPackage}";
+            AssetDatabase.ImportPackage(filePackagePath, interactive: true);
+        }
+
+        [MenuItem("Amazon GameLift/Help/Documentation", priority = 200)]
+        public static void OpenDocumentation()
+        {
+            Application.OpenURL(Urls.AwsHelpGameLiftUnity);
+        }
+
+        [MenuItem("Amazon GameLift/Help/AWS GameTech Forum", priority = 201)]
+        public static void OpenGameTechForums()
+        {
+            Application.OpenURL(Urls.AwsGameTechForums);
+        }
+
+        [MenuItem("Amazon GameLift/Help/Report Issues", priority = 202)]
+        public static void OpenReportIssues()
+        {
+            Application.OpenURL(Urls.GitHubAwsLabs);
+        }
+
+        private void CreateGUI()
+        {
+            _root = rootVisualElement;
+            _mVisualTreeAsset = UnityEngine.Resources.Load<VisualTreeAsset>("EditorWindow/GameLiftPlugin");
+            if (_mVisualTreeAsset == null)
+            {
+                return;
+            }
+
+            VisualElement uxml = _mVisualTreeAsset.Instantiate();
+            _root.Add(uxml);
+            
+            ApplyText();
+
+            var contentContainer = _root.Q(className: "main__content");
+
+            _tabButtons = _root.Query<Button>(className: TabButtonClassName).ToList();
+            _tabContent = _root.Query(className: TabContentClassName).ToList();
+
+            _tabButtons.ForEach(button => button.RegisterCallback<ClickEvent>(_ => { OpenTab(button.name); }));
+        }
+
+        private void ApplyText()
+        {
+            SetElementText("Landing", Strings.TabLanding);
+            SetElementText("Credentials", Strings.TabCredentials);
+            SetElementText("Anywhere", Strings.TabAnywhere);
+            SetElementText("EC2", Strings.TabEC2);
+            SetElementText("Help", Strings.TabHelp);
+        }
+
+        private void SetElementText(string elementName, string text)
```

Comment:
Same as above comment, this should be pulled out into a separate utility (class) which allows performing such graphical operations